src/analytics/private/metrics-common-workflow.ts

```diff
@@ -80,6 +85,15 @@ export function buildMetricsWidgetForWorkflows(scope: Construct, id: string, pro
   });
   (upsertUsersWorkflowAlarm.node.defaultChild as CfnResource).addPropertyOverride('Period', upsertUsersInterval.getIntervalSeconds());
 
+  const scanMetadataWorkflowAlarm = new Alarm(scope, id + 'ScanMetadataWorkflowAlarm', {
+    comparisonOperator: ComparisonOperator.GREATER_THAN_OR_EQUAL_TO_THRESHOLD,
+    threshold: 1,
+    evaluationPeriods: 1,
+    treatMissingData: TreatMissingData.NOT_BREACHING,
+    metric: props.scanMetadataWorkflow.metricFailed({ period: Duration.hours(1) }),
```

Comment:
need  addPropertyOverride the `period` and `threshold` based on processing ijb interval.

Reply:
Override period with processing job interval